test/libyul/YulInterpreterTest.cpp

```diff
@@ -94,13 +94,13 @@ string YulInterpreterTest::interpret()
 	state.maxExprNesting = 64;
 	try
 	{
-		Interpreter::run(state, EVMDialect::strictAssemblyForEVMObjects(langutil::EVMVersion{}), *m_ast);
+		Interpreter::run(state, EVMDialect::strictAssemblyForEVMObjects(langutil::EVMVersion{}), *m_ast, false);
```

Comment:
Memory tracing is enabled (last argument = `false`) for Yul interpreter tests. This may be changed to `true` if tests pass only without memory tracing.